classes/Language.php

```diff
@@ -188,7 +188,11 @@ public function add($autodate = true, $nullValues = false, $only_add = false)
         }
 
         if ($this->is_rtl) {
-            self::installRtlStylesheets(true, false, null, null, (defined('PS_INSTALLATION_IN_PROGRESS') ? true : false));
+            self::getRtlStylesheetProcessor()
+                ->setIsInstall((defined('PS_INSTALLATION_IN_PROGRESS') ? true : false))
```

Comment:
You can remove the ternary test as you will get the boolean you're looking for.

Reply:
True, I'll change that